src/mats_l1_processing/instrument.py

```diff
@@ -84,14 +84,14 @@ class CCD:
 
     """
 
-    def __init__(self, channel, calibration_file):
+    def __init__(self, channel, calibration_file, start_datetime: datetime = datetime(2000,1,1), end_datetime: datetime = datetime(3000,1,1)):
 
         """Init method for CCD class
 
         Args:
             channel (str): Name of channel
             calibration_file (str): calibration file containing paths to calibration files
-
+            date (datetime): datetime on which to extract calibration parameters (default: None) 
```

Comment:
This is not what the signature of `__init__` says.

Reply:
fixed